src/app/router/connectorRoutes.tsx

```diff
@@ -61,25 +58,15 @@ function getConnectorRoutes() {
       title: i18n.t("translation:choose_connector.lnbits.title"),
       logo: lnbits,
     },
-    {
-      path: "lnd-hub-go",
-      element: <ConnectLndHub lndHubType="lndhub_go" />,
-      title: i18n.t("translation:choose_connector.lndhub_go.title"),
-      logo: lndhubGo,
-    },
+
     {
       path: "kollider",
       element: <ConnectKollider />,
       title: i18n.t("translation:choose_connector.kollider.title"),
       description: i18n.t("translation:choose_connector.kollider.description"),
       logo: kolliderLogo,
     },
-    {
-      path: "lnd-hub-bluewallet",
-      element: <ConnectLndHub />,
-      title: i18n.t("translation:choose_connector.lndhub_bluewallet.title"),
-      logo: lndhubBlueWallet,
```

Comment:
Can we remove the bluewallet logo as well or is that used somewhere else?

Reply:
okay sir, i'll remove it

Reply:
all changes has been made, thanks